modules/rhel-compute-updating.adoc

```diff
@@ -25,6 +29,8 @@ your RHEL machines and the `upgrade` playbook.
 
 .Procedure
 
+ifndef::restricted[]
```

Comment:
For upgrade in restricted network, we still need to inform user to create yum repo files which are pointing to the newly synced mirror repo in the above section to instead of subscrption-manager repos.

Reply:
Need some word polish here,  in a restricted network, subscription-manager is not working well, that is why we have `Obtaining required software packages and images` and `Prepare and populate the repository server` section, we need to inform user to use the created internal repository to replace subscrption-manager commands.